costmap_2d/include/costmap_2d/layer.h

```diff
@@ -64,12 +64,48 @@ class Layer
   virtual void updateBounds(double robot_x, double robot_y, double robot_yaw, double* min_x, double* min_y,
                             double* max_x, double* max_y) {}
 
+  /**
+   * @brief Modified updateBounds() to use time argument t for timed layers. 
+   *        Calls the above function without t by default.
+   */
+  virtual void updateBounds(double robot_x, double robot_y, double robot_yaw, double* min_x, double* min_y,
+                            double* max_x, double* max_y, double t) {
+                              updateBounds(robot_x, robot_y, robot_yaw, min_x, min_y, max_x, max_y);}
```

Comment:
S: fix formatting for this function and the one below.
Try in vscode to right click -> format selection